src/components/App.js

```diff
@@ -10,9 +10,11 @@ import MovieDetails from './MovieDetails'
 function App() {
   return (
     <Router>
-      <Route path="/" component={HeaderNav} />
+      <Switch>
+        <Route exact path="/movies/:id" component={MovieDetails} />
+        <Route path="/" component={HeaderNav} />
```

Comment:
What if we change this component to "ContentPage" (or something similar, names are hard ><). Inside of that we would have
1. The header
2. Another router, that for now would only have the HomePage?

One of the cool (but also confusing) things about react router is you can nest routers inside of a component.

Reply:
Done